docs/theming/developers.md

```diff
@@ -200,46 +207,47 @@ themable graphics
 
 ## How theming works
 
-We developed our theming system with web standards in mind, aiming for it to use
-CSS over JavaScript as much as possible. At the current state of the art, we
-found that some JavaScript is required for the system to work correctly, but in 
-the near  future, we expect to be able to remove all or nearly all of that 
-JavaScript, and have the theming system work entirely (or almost entirely) 
+We developed our theming system with web standards in mind, aiming to use
+CSS over JavaScript as much as possible. At the current state of browser technology,
+we found that some JavaScript is required for the system to work correctly.
+But shortly, we expect to be able to remove all or nearly all of that 
```

Comment:
`shortly, we` → `we soon`

`all or nearly all of` → `all (or nearly all) of`
_This matches the "entirely (or almost entirely)" parenthetical in the next line._

Reply:
[fe891c8](https://github.com/RedHat-UX/red-hat-design-system/pull/1937/commits/fe891c8237167f995351267d3e97915950f26d3a)